docs/source/puppet/whatisthis.xml

```diff
@@ -1 +1,2 @@
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
+<mxfile host="www.draw.io" modified="2019-11-25T09:49:02.403Z" agent="Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:70.0) Gecko/20100101 Firefox/70.0" version="12.2.9" etag="XNW1qxzD3Zf6k0rfsNaP" type="device" pages="1"><diagram id="dcb8233a-4aa4-e58c-2717-36c312f25f96" name="Page-1">7VrLbts6EP0abwNKsmVr6ThJWyAFgnpRdFUwEi3xltIIFBXb/fqSFqkX5SRA5Dg3cLKIOeLznDOjGcYTb5XuvnCcJ98hImziomg38W4mrutM0UL+UZZ9ZZl7qDLEnEa6U2NY079EG023kkak6HQUAEzQvGsMIctIKDo2zDlsu902wLqr5jgmlmEdYmZbf9JIJJV1MUON/SuhcWJWdpB+kmLTWRuKBEewbZm824m34gCi+pTuVoQp8Awu1bi7I0/rjXGSidcM8KoBT5iV+mx6X2JvDiu3mKuP/IDkdYSLhKjRjmwUgsMfsgIG/NDZQ2ge3PjyyYYyZuwZZESZIBOaS2eq292hi5u5tGNG40zaQnkGIh9eJyJlekG1+gMW0p4dLC5SM9mn1kA8ES7IrmXSKHwhkBLB97KLeYqCaoiWpGs0um0IdgPNWtIi1zXcYi2quJ67wV1+0NAP0+BaNPwgMYXMIgNKwWgmETfCRvL4MccRJQ2YGu8WbNuECrLOcaim2Uqn7NHhmrZeSc1qeE93sfLiK7wtplcxhzI/LPlNOtfg09+82vqAOhbzwAmOqENRRaWLLTX7AvKWFhjZKPEV8gg0i+8PrRsP2Sqql+jodAyBuD2BuAMCmQ4IxPFHEMjUEsjX8lEalmEIpTxTXyfyUKIrgS4ZxiVtHizv6xOT0ihSywyKSgogiw6wo9P4pTO3YQ+cIbd8O+izS3DU6vO7JPgzW/qzU8VG32JhnUtc1ayfXfx93D0bdwedSPzzi/iPkGCSpndR/+Ki/vqNG9jAn0z+A3InkSwCdBO4SCCGDLPbxtqDoMXAf0SIvVY4LgVIUzPDPahU5/k8pYCSh3ofGhWBeUzqHKOyqS0+Cy0nDAv61K1jhnA6DF1yjvetDjnQTBStmR+UocWYqeOMqzheG/MX+ztT1OOo2kHDWH2U15Ho/a9InH0KEk11PxaJgRUBlz/X0rCCiDzQnKh6yKJ5sE7qvVbk751a8lj9ZL3CNoefXuxsT1S/AVEreN7jR8IeoKCCQieI9qPrIwgB6QuvuZcqOXNOo3GnqZpkyztW1+UcojIUVVF3nQ9Wd6EEPDeAjxLh3X6En9nJ/WKopHJGiPDOxw0OjjsQHdDY0eHVQNm3E7+W3+8t9Iak2c5Cerp7xAUNryIIy5SYNOZtanK7YvLtdMEfEtMYaZoz/bhiQraYgrNpCT0TzVeQptROaC+x/DSxPKzgHueaZnrGSF6r/gM638x2Pvd8kdy+0aq8b034Ew1VVbnCAjOIL074Dk5YVKj/DjXm41TNQf/OyH4Nni6nWnxcT/QHcqr52TzRvtW8eOIn98TgPYsb+/pW51kMyugOeIrFq/+/eBHX23ItBfmmhnwUbfkuOpu23KM5/EVbn1Fbp8wgZLP5skt1Odh8Zci7/Qc=</diagram></mxfile>
```

Comment:
@VictorRaim I would recommend you to use the AWS-style boxes for accounts like this instead : 
![Screenshot from 2019-11-26 11-41-13](https://user-images.githubusercontent.com/37543966/69622723-ceffa300-1041-11ea-97bf-5767860f03b4.png)

Also, the `Spoke account` concept could be clarified by having three occurrence of them, named respectively `Spoke account 1`, `spoke account 2`, `spoke account N` to make clear that there is no conceptual upper limits for the number of spoke accounts